onboard/nucleo_bridge/LCMHandler.cpp

```diff
@@ -0,0 +1,188 @@
+    {
+        internal_object->refreshAngles();
+        internal_object->sa_pos_data();
+        internal_object->ra_pos_data();
+    }
+}
+
+//The following functions are handlers for the corresponding lcm messages
+void LCMHandler::InternalHandler::ra_closed_loop_cmd(LCM_INPUT, const ArmPosition *msg)
+{
+    ControllerMap::controllers["RA_0"]->closed_loop(0, msg->joint_a);
+    ControllerMap::controllers["RA_1"]->closed_loop(0, msg->joint_b);
+    ControllerMap::controllers["RA_2"]->closed_loop(0, msg->joint_c);
+    ControllerMap::controllers["RA_3"]->closed_loop(0, msg->joint_d);
+    ControllerMap::controllers["RA_4"]->closed_loop(0, msg->joint_e);
+    ControllerMap::controllers["RA_5"]->closed_loop(0, msg->joint_f);
+    ra_pos_data();
+}
+
+void LCMHandler::InternalHandler::sa_closed_loop_cmd(LCM_INPUT, const SAClosedLoopCmd *msg)
+{
+    ControllerMap::controllers["SA_0"]->closed_loop(msg->torque[0], msg->angle[0]);
+    ControllerMap::controllers["SA_1"]->closed_loop(msg->torque[1], msg->angle[1]);
+    ControllerMap::controllers["SA_2"]->closed_loop(msg->torque[2], msg->angle[2]);
+    sa_pos_data();
+}
+
+void LCMHandler::InternalHandler::ra_open_loop_cmd(LCM_INPUT, const RAOpenLoopCmd *msg)
+{
+    ControllerMap::controllers["RA_0"]->open_loop(msg->throttle[0]);
+    ControllerMap::controllers["RA_1"]->open_loop(msg->throttle[1]);
+    ControllerMap::controllers["RA_2"]->open_loop(msg->throttle[2]);
+    ControllerMap::controllers["RA_3"]->open_loop(msg->throttle[3]);
+    ControllerMap::controllers["RA_4"]->open_loop(msg->throttle[4]);
+    ControllerMap::controllers["RA_5"]->open_loop(msg->throttle[5]);
+    ra_pos_data();
+}
+
+void LCMHandler::InternalHandler::sa_open_loop_cmd(LCM_INPUT, const SAOpenLoopCmd *msg)
+{
+    ControllerMap::controllers["SA_0"]->open_loop(msg->throttle[0]);
+    ControllerMap::controllers["SA_1"]->open_loop(msg->throttle[1]);
+    ControllerMap::controllers["SA_2"]->open_loop(msg->throttle[2]);
+    sa_pos_data();
+}
+
+/*void LCMHandler::InternalHandler::ra_config_cmd(LCM_INPUT, const RAConfigCmd *msg)
+{
+    ControllerMap::controllers["RA_0"]->config(msg->Kp[0], msg->Ki[0], msg->Kd[0]);
+	    ControllerMap::controllers["RA_1"]->config(msg->Kp[1], msg->Ki[1], msg->Kd[1]);
+	    ControllerMap::controllers["RA_2"]->config(msg->Kp[2], msg->Ki[2], msg->Kd[2]);
+	    ControllerMap::controllers["RA_3"]->config(msg->Kp[3], msg->Ki[3], msg->Kd[3]);
+	    ControllerMap::controllers["RA_4"]->config(msg->Kp[4], msg->Ki[4], msg->Kd[4]);
+	    ControllerMap::controllers["RA_5"]->config(msg->Kp[5], msg->Ki[5], msg->Kd[5]);
+}
+
+void LCMHandler::InternalHandler::sa_config_cmd(LCM_INPUT, const SAConfigCmd *msg)
+{
+    ControllerMap::controllers["SA_0"]->config(msg->Kp[0], msg->Ki[0], msg->Kd[0]);
+    ControllerMap::controllers["SA_1"]->config(msg->Kp[1], msg->Ki[1], msg->Kd[1]);
+    ControllerMap::controllers["SA_2"]->config(msg->Kp[2], msg->Ki[2], msg->Kd[2]);
+}*/

```

Comment:
Please remove commented out code throughout this PR

Reply:
This could be important for when we activate IK on the 2021 arm